cli/ops.rs

```diff
@@ -620,6 +622,46 @@ fn op_env(
   ))
 }
 
+fn op_deps(
+  state: &ThreadSafeState,
+  base: &msg::Base<'_>,
+  data: Option<PinnedBuf>,
+) -> Box<OpWithError> {
+  assert!(data.is_none());
+  let cmd_id = base.cmd_id();
+
+  let maybe_out = worker::fetch_module_meta_data_and_maybe_compile(
+    &state,
+    &state.main_module().unwrap(),
```

Comment:
I think you can easily take it from message to allow to inspect any file:
```
Deno.deps("https://deno.land/log/mod.ds");
```

Reply:
> I think you can easily take module name from message to allow to inspect any file:

I'm thinking about it, but it's not obvious to me how to find out which module as called an op... It seems this is some functionality we need in core.